### DISCUSSION TITLE
Program or command to run 16-bit Windows app maximized?

### DISCUSSION THREAD

**Discussion by emendelson** [otya128/winevdm#1370] [Q&A]
I have a 16-bit Windows app that always starts with a 640X480 window. I can click the up arrow to maximize it, but I'm trying to find a way to maximize it automatically. I seem to remember that there was an old PCMag utility that would maximize a program on startup, but I can't find it, and I may be imagining it. Is there anything out there that I can use for this?

**Comment by cracyc** [comment]
If you create a shortcut to the program then you can set maximized in the run option of the shortcut properties window.

**Comment by emendelson** [comment]
That was the first thing I tried. Unfortunately, the program seems to be hard-wired to open at VGA resolution, though it can then be maximized with the up arrow. I also found an old Win31 app called StartIt that replaces shell.dll, but that doesn't work with otvdm. I don't think there's an easy solution to this one, and I live without finding one...
